src/main/modules/oidc/index.ts

```diff
@@ -51,13 +51,16 @@ export class OidcMiddleware {
       errorHandler(async (req: AppRequest, res: Response, next: NextFunction) => {
         console.log('inside app.use');
         console.log('req.path is ' + req.path);
+        if (app.locals.developmentMode) {
+          req.session.c100RebuildLdFlag = config.get('launchDarkly.offline');
+        }
         const c100RebuildLdFlag: boolean =
           req.session.c100RebuildLdFlag !== undefined
             ? req.session.c100RebuildLdFlag
             : (req.session.c100RebuildLdFlag = await getFeatureToggle().isC100reBuildEnabled());
-
+        console.log('C100 - Launch Darkly Flag ', c100RebuildLdFlag);
         //Skipping for C100 rebuild
-        if (req.path.startsWith(CITIZEN_HOME_URL || C100_URL || DASHBOARD_URL) && !req.session?.user) {
+        if (req.path.startsWith(C100_URL)) {
```

Comment:
@yogendraU I think it will only cases for c100 rebuild & skip for private law ?

Reply:
Thisi s needed only for C100. If C100 URL is in the path, we need to check the LD flag.